changes/ce/fix-14099.en.md

```diff
@@ -0,0 +1,5 @@
+Fix the issue that `esockd` prints a supervisor report on `malformed_utf8_string_length` error:
```

Comment:
Removed a error level log when failed to validate UTF-8 strings in MQTT messages.